Shared/Samples/Identify layer features/IdentifyLayerFeaturesView.swift

```diff
@@ -0,0 +1,141 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct IdentifyLayerFeaturesView: View {
+    /// A map with a topographic basemap centered on the United States.
+    @State var map: Map = {
+        let map = Map(basemapStyle: .arcGISTopographic)
+        map.initialViewpoint = Viewpoint(
+            center: Point(x: -10977012.785807, y: 4514257.550369, spatialReference: .webMercator),
+            scale: 68015210
+        )
+        return map
+    }()
+    
+    /// The tapped screen point.
+    @State var tapScreenPoint: CGPoint?
+    
+    /// The string text for the identify layer results overlay.
+    @State var overlayText = "Tap on the map to identify feature layers."
+    
+    /// A Boolean value indicating whether to show an error alert.
+    @State var isShowingAlert = false
+    
+    /// The error shown in the error alert.
+    @State var error: Error? {
+        didSet { isShowingAlert = error != nil }
+    }
+    
+    var body: some View {
+        ZStack {
+            MapViewReader { proxy in
+                MapView(map: map)
+                    .onSingleTapGesture { screenPoint, _ in
+                        tapScreenPoint = screenPoint
+                    }
+                    .task {
+                        do {
+                            // Add map image layer to the map.
+                            let mapImageLayer = ArcGISMapImageLayer(url: .worldCities)
+                            try await mapImageLayer.load()
+                            
+                            // Hide continent and world layers.
+                            mapImageLayer.subLayerContents[1].isVisible = false
+                            mapImageLayer.subLayerContents[2].isVisible = false
+                            map.addOperationalLayer(mapImageLayer)
+                            
+                            // Add feature layer to the map.
+                            let featureTable = ServiceFeatureTable(url: .damageAssessment)
+                            try await featureTable.load()
+                            let featureLayer = FeatureLayer(featureTable: featureTable)
+                            map.addOperationalLayer(featureLayer)
+                        } catch {
+                            // Present error load the layers if any.
+                            self.error = error
+                        }
+                    }
+                    .task(id: tapScreenPoint) {
+                        // Identify layers using the screen point.
+                        if let screenPoint = tapScreenPoint,
+                           let results = try? await proxy.identifyLayers(
+                            screenPoint: screenPoint,
+                            tolerance: 12,
+                            returnPopupsOnly: false,
+                            maximumResultsPerLayer: 10
+                           ) {
```

Comment:
It looks like the indentation is off here, can you auto re-indent?

Reply:
That is actually where the auto indent puts it.